test/argent-account-test.ts

```diff
@@ -44,10 +44,13 @@ describe("Argent account", function () {
         });
         argentAccountAddress = account.address;
 
+        const estimatedFee = await account.estimateDeployAccountFee();
```

Comment:
Since it's not really important for this test (we are relying on the implicit estimation), let's extract this to a separate test. Same for oz account.

Reply:
Also, we have a utility function which asserts fee estimation format, that could be used here.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/96/commits/22f550e7a64fffbcd7ec9f5c509f1a30ae2e8e60